### ISSUE TITLE
CDs not recognized

### ISSUE THREAD

**Issue by JustJP2K** [libretro/RetroArch#16637]
## Description

When I insert a game disk and click on load disk it will say failed to load content

### Expected behavior

Insert CD, gets read and game starts like PCSX2

### Actual behavior

CD gets read, error no items

### Steps to reproduce the bug

Use PCSX2 core
Insert Game CD into CD drive
Click on load CD

**Comment by hizzlekizzle** [comment]
Support for real-disk loading has to be added to the individual cores, AFAIK. In the case of PCSX2, only a handful of games are CD-based.

**Comment by JustJP2K** [comment]
PCSX2 supports CD read on it's own. But when using the core in retroarch it doesn't work.

**Comment by hizzlekizzle** [comment]
Yes, that's correct, but to do that, the core would either need direct access to the hardware (and have that part activated/hooked up in the core) or it needs to be modified to access the CD drive through RetroArch/libretro.

**Comment by JustJP2K** [comment]
I see. Could that perhaps be added to retroarch then? To me it seems to be a vital part of any emulator to have CD read capabilities for those who still have all the old game CDs and don't want to rip it into an iso file.
